### DISCUSSION TITLE
Welcome to FlexGet Discussions!

### DISCUSSION THREAD

**Discussion by gazpachoking** [Flexget/Flexget#2814] [General]
## 👋 Welcome!

We've had some trouble with the host of our old forum, and are using this as an opportunity to try out GitHub Discussions.
Feel free to use this area to ask and answer questions, and provide feedback on the forum platform itself.
Sorry about the abrupt removal of the old forum!

To post configs or logs, surround them with triple backticks like this:

    ```
    tasks:
      blah:
        plugin: whatever
    ```

**Comment by cvium** [comment]
It would be nice with a read-only archive. There's a lot of information in that forum.

**Comment by gazpachoking** [comment]
Well... I didn't take the forum offline by choice. Unfortunately I wasn't keeping regular backups of the forum, and it turns out, neither was the host I was paying to do it. They let the RAID fail, then after a day of telling me they were restoring backups, revealed that their backup script was not backing up our VPS. I do have a backup from a year ago, but would need to find alternate hosting, as I've cancelled that one, obviously. 😞 Sorry everyone.

**Reply by cvium** [reply]
If it can be converted to static pages somehow, we could just host it on GH pages.

**Reply by lamski** [reply]
A static, read-only view of the old forum pages would be nice.

If there are plans to setup a new forum server, perhaps it would be nice to explore the possibility of setting it up like StackOverflow?
Most of the stuff in the forum are in the question-and-answer format anyway. The StackOverflow format would fit perfectly. Users being able to vote and tag posts would make it easier to find answers to technical questions and organically group/segregate posts according to topics. People getting brownie points for getting their answers accepted might give people incentive to help out.

**Reply by gazpachoking** [reply]
If GitHub Discussions works well enough, I'd like to stick with this, as: it's free, hosted for us, and reliable.

**Reply by frankyw** [reply]
Github discussions is great!

**Reply by specie77** [reply]
Hi @gazpachoking 

Having some trouble with exists_movies not reading my existing library when rebuilding the database. Google thinks the following discussion link would be useful but based on the above info and the failure of the hosting provider, I can't get to it anymore. Can you dig this discussion thread out of your backup?

https://discuss.flexget.com/t/help-configuration-movie-download/3388/3

Thanks!
